rafthttp/http.go

```diff
@@ -72,6 +74,15 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if err := checkVersionCompability(serverVersion(r.Header), minClusterVersion(r.Header)); err != nil {
+		log.Printf("rafthttp: request ignored due to %v", err)
+		http.Error(w, errMsgIncompatibleVersion, http.StatusForbidden)
+		return
+	}
+	if err := checkClusterVersionCompatiblity(r.Header.Get("X-Cluster-Version")); err != nil {
+		log.Printf("rafthttp: received request that %v", err)
```

Comment:
The local version is 2.1, the remote version is 2.2, and the cluster version is 2.0. The local can talk to the remote, but the remote may not be able to talk to some member in the cluster(or he just needs to wait some time for version upgrade).